res/css/photon-components.css

```diff
@@ -35,10 +35,43 @@
   margin-left: 8px;
 }
 
-.photon-button:hover:active {
+.photon-button:active {
   background-color: var(--grey-90-a30);
 }
 
+.photon-button-primary,
+.photon-button-default {
+  min-width: 132px;
+  height: 32px;
+  margin: 0 8px;
+  font-size: 13px;
+}
+
+.photon-button-default {
+  background-color: var(--grey-90-a10);
+}
+
+.photon-button-default:hover {
+  background-color: var(--grey-90-a20);
+}
+
+.photon-button-default:active {
+  background-color: var(--grey-90-a30);
+}
```

Comment:
I think we don't need `.photon-button-default`, it can just be the default with `.photon-button` (This is already the case for the colors BTW).
The only thing I missed when I did it originally is about the height, but I really think this should go to `photon-button` as well, and we should look at the few places where we already used this class and possibly override it.

Reply:
I'm keeping the photon-button-default class name, as it implements more specific stylings for the larger button. However, I removed some of the duplicate color definitions here.